wemake_python_styleguide/visitors/ast/classes.py

```diff
@@ -69,3 +70,20 @@ def visit_any_function(self, node: AnyFunctionDef) -> None:
         self._check_decorators(node)
         self._check_magic_methods(node)
         self.generic_visit(node)
+
+
+class WrongParentClassListVisitor(BaseNodeVisitor):
```

Comment:
Why do we need a new class for this check? Can we just extend an existing one? `WrongClassVisitor`?